### DISCUSSION TITLE
[Question] Is it possible for the options to open in a modal?

### DISCUSSION THREAD

**Discussion by quriouss** [MobileReality/react-native-select-pro#87] [Q&A]
Hi! Just started using this package today and I find it really promising, GJ guys!
But I have a question. Title says it all. Can it be done without any custom modifications?

Example:
![image](https://user-images.githubusercontent.com/108438295/187236263-c845640c-c369-4514-8a41-f90228f40bb0.png)

**Comment by irekrog** [comment]
Hi @quriouss 

Do you mean open a dropdown inside a modal? If so you can do it with `SelectModalProvider` https://mobilereality.github.io/react-native-select-pro/docs/basic-usage

**Reply by quriouss** [reply]
Hi! Thanks for your reply!

No. I would like the `Select` component to be rendered in a screen like this: 
<img width="81" alt="Screenshot 2022-08-30 at 12 57 44 PM" src="https://user-images.githubusercontent.com/108438295/187408275-ae6bd461-c72b-4d4a-a90e-bfa3e894e6f8.png">
but when I click on it, I would like the options to be shown in a modal instead of a dropdown.

The example you sent me is about rendering the whole `Select` component in a modal.

**Reply by irekrog** [reply]
No, at this moment we don't support this feature. Maybe in the future we will add this
